gcc/brig/brigfrontend/brig-util.h

```diff
@@ -76,4 +86,25 @@ bool gccbrig_might_be_host_defined_var_p (const BrigDirectiveVariable *brigVar);
 /* From hsa.h.  */
 bool hsa_type_packed_p (BrigType16_t type);
 
+struct reg_use_info
+{
+  /* The generic tree types as the registers is interpreted and their
```

Comment:
This comment is a bit unclear. Also GCC style requires two spaces after dots.

Something like this would be clearer:
"This vector keeps count of the times an HSAIL register is used for storing data of a given type..."